client/ayon_blender/plugins/create/create_model.py

```diff
@@ -24,8 +22,8 @@ def create(
 
         # Add selected objects to instance
         if pre_create_data.get("use_selection"):
-            bpy.context.view_layer.objects.active = asset_group
-            for obj in lib.get_selection():
-                obj.parent = asset_group
+            selections = lib.get_selection()
+            top_root = lib.get_highest_root(selections)
+            top_root.parent = asset_group
```

Comment:
Please revert this change.

This should just be:
```python
for obj in selection:
    obj.parent = asset_group
```

The user selects, without hierarchy, what they want to add into the asset group. If they happen to select children as well, then that will also get parented to the asset group - that's absolutely fine behavior. They'll learn and see that if that's not what they want, then don't select the children - select just the root, etc.